### PR TITLE
add pre-reader section, and customize projects widget for young students

### PR DESCRIPTION
**Pull request by davidsbailey**
This addresses the following work items on the ProjectWidget from the [spec](https://docs.google.com/document/d/1NYgN4G3FDlyTevKGE4_5wNzYkB36BJKZzsLn_ESLGmA/edit#heading=h.ygy1ah19vtrq):
* add pre-reader section
* hide *labs when isTooYoung is specified

caveats:
* isTooYoung and canViewFullList are not hooked up to anything yet, so you can only see them in storybook

screencast:
project-widget-istooyoung

### CODE REVIEW

File: apps/src/templates/projects/NewProjectButtons.jsx [hunk 1]
```diff
@@ -12,21 +12,28 @@ const DEFAULT_PROJECT_TYPES = [
...
+];
+
 const PROJECT_INFO = {
   'playlab': {
     label: i18n.projectTypePlaylab(),
     thumbnail: "/shared/images/fill-70x70/courses/logo_playlab.png"
   },
   'playlab_k1': {
-    label: i18n.projectTypePlaylab(),
+    label: i18n.projectTypePlaylabPreReader(),
```
Comment (caleybrock) [inline apps/src/templates/projects/NewProjectButtons.jsx:29]: I'd check with Ryan on these. For the project widget in stage extras, levelbuilders would choose between the two level types but they were both supposed to both say "Play Lab", but it's possible he's fine with this change.

File: apps/src/templates/projects/NewProjectButtons.jsx [hunk 2]
```diff
@@ -150,13 +157,15 @@ class NewProjectButtons extends React.Component {
...
     isRtl: PropTypes.bool,
     description: PropTypes.string,
+    isTooYoung: PropTypes.bool,
   };
 
   render() {
-    const { isRtl, description } = this.props;
+    const { isRtl, isTooYoung, description } = this.props;
     const thumbnailStyle = isRtl ? styles.thumbnailRtl : styles.thumbnail;
     // Using absolute URLs to get this working in storybook.
```
Comment (caleybrock) [inline apps/src/templates/projects/NewProjectButtons.jsx]: I know this isn't in your PR, but this comment seems out of place.

File: apps/src/templates/projects/ProjectWidget.jsx [hunk 3]
```diff
@@ -29,6 +29,7 @@ class ProjectWidget extends React.Component {
     isLoading: PropTypes.bool,
     isRtl: PropTypes.bool,
     canViewFullList: PropTypes.bool,
+    isTooYoung: PropTypes.bool,
```
Comment (caleybrock) [inline apps/src/templates/projects/ProjectWidget.jsx]: let's call this `canViewAdvanced`

File: apps/src/templates/projects/NewProjectButtons.jsx (hunk 1 above)
Comment (davidsbailey) [reply apps/src/templates/projects/NewProjectButtons.jsx:29]: I've got confirmation with Ryan that these strings should have "Pre-reader" on them going forward.

**Comment by davidsbailey** [comment]
Thanks Caley! ready for another look

File: apps/src/templates/projects/NewProjectButtons.jsx [hunk 4]
```diff
@@ -4,29 +4,37 @@ import styleConstants from '../../styleConstants';
...
 
-const DEFAULT_PROJECT_TYPES = [
+const DEFAULT_PROJECT_TYPES_ADVANCED = [
   'playlab',
   'artist',
   'applab',
   'gamelab'
 ];
 
+const DEFAULT_PROJECT_TYPES_BASIC = [
```
Comment (caleybrock) [inline apps/src/templates/projects/NewProjectButtons.jsx:16]: this naming is much better 👍

**Pull request merged**